examples/07-collaboration/06-ghost-writer/.bnexample.json

```diff
@@ -0,0 +1,10 @@
+{
+  "playground": true,
+  "docs": true,
```

Comment:
I don't think this should be listed in docs - it's not a useful code snippet to learn from, right?

Reply:
Didn't know that was possible!